src/styles/utils/_mixins.scss

```
@@ -0,0 +1,5 @@
+@mixin screen-size ($width, $columns) {
+  @media screen and (min-width:$width) {
+  grid-template-columns: repeat($columns, 200px);
+}
+}
```

```suggestion
  }
}
```
I didn't even think about using mixins for media:)